doc/howto/cluster_config_networks.md

```diff
@@ -1,2 +1,60 @@
 (cluster-config-networks)=
 # How to configure networks for a cluster
+
+As mentioned above, all nodes must have identical networks defined.
+
+The only difference between networks on different nodes might be their optional configuration keys.
```

Comment:
Quite a few mention of nodes rather than members here

Reply:
That is not the latest commit!
But the commit that moves the content without changes. That's also why the page starts with "As mentioned above".

Reply:
Its tricky to review when I dont know which is the final form.